scripts/sanitycheck

```diff
@@ -2688,6 +2688,8 @@ class TestSuite:
             for row in cr:
                 if row["arch"] == "arch":
                     continue
+                if row["passed"] == "skipped":
```

Comment:
it is not a good idea to have boolean values mixed with literals. I do not like the binary column called passed, I'd rather change this column completely and call it status (we have that already) and put the right status in it. Let me spend some time and clean this up for real. I thought I fixed this already, because it did annoy me at some point, seems like I missed it.

Reply:
Sounds good.  I'd prefer to rename the field to status as well!